doc/api/tls.md

```diff
@@ -1403,11 +1470,13 @@ where `secureSocket` has the same API as `pair.cleartext`.
 [`tls.Server`]: #tls_class_tls_server
 [`tls.TLSSocket.getPeerCertificate()`]: #tls_tlssocket_getpeercertificate_detailed
 [`tls.TLSSocket`]: #tls_class_tls_tlssocket
+[`tls.checkServerIdentity()`]: #tls_tls_checkserveridentity_hostname_cert
 [`tls.connect()`]: #tls_tls_connect_options_callback
 [`tls.createSecureContext()`]: #tls_tls_createsecurecontext_options
 [`tls.createSecurePair()`]: #tls_tls_createsecurepair_context_isserver_requestcert_rejectunauthorized_options
 [`tls.createServer()`]: #tls_tls_createserver_options_secureconnectionlistener
 [`tls.getCiphers()`]: #tls_tls_getciphers
+[`tls.TLSSocket.verifyError()`]: #tls_tlssocket_verifyerror
```

Comment:
Nit: needs no be after `` [`tls.TLSSocket`]: #tls_class_tls_tlssocket `` ASCII-wise.